packages/openapi-generator/src/file-serializer/schema.ts

```diff
@@ -110,5 +115,25 @@ function serializePropertyWithDocumentation(
 export function schemaPropertyDocumentation(
   schema: OpenApiObjectSchemaProperty
 ): string {
-  return schema.description ? documentationBlock`${schema.description}` : '';
+  const signature: string[] = [];
+  if (schema.description) {
+    signature.push(schema.description);
+  }
+
+  if (schema.schema?.deprecated) {
+    signature.push('@deprecated');
+  }
+
+  if (schema.schema?.example) {
+    signature.push(`@example ${JSON.stringify(schema.schema?.example)}`);
```

Comment:
Cool idea, I would not have come up with this for values that are not JSON. But I checked, it works.

This will make it look even nicer:
```suggestion
    signature.push(`@example${unixEOL}${JSON.stringify(schema.schema?.example, null, 2)}`);
```

Reply:
Adding EOL formats it like 
/**
  * @example
  * 1.2
  **/
as opposed to in the same line which looks more consistent with how other comments are formatted.
/**
  * @example 1.2
  **/

Do you think it makes more sense like this or I should move it to the next line?

Reply:
Both are fine 😊, ideally I would put short examples on the same line (e. g. less than 20 characters or so) and long examples below. But I also think this doesn't matter.
https://tsdoc.org/pages/tags/example/